provider/resources.go

```diff
@@ -800,6 +801,7 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_cognito_user_pool":                      {Tok: awsResource(cognitoMod, "UserPool")},
 			"aws_cognito_user_pool_client":               {Tok: awsResource(cognitoMod, "UserPoolClient")},
 			"aws_cognito_user_pool_domain":               {Tok: awsResource(cognitoMod, "UserPoolDomain")},
+			"aws_cognito_user_pool_ui_customization":     {Tok: awsResource(cognitoMod, "UserPoolUiCustomization")},
```

Comment:
It's usually prescribed to keep `UI` in caps. I'd try that unless it screws up anything.